lib/Expression/Parselet/NameParselet.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace PhpBench\Expression\Parselet;
+
+use PhpBench\Expression\Ast\Node;
+use PhpBench\Expression\Ast\ParameterNode;
+use PhpBench\Expression\Parser;
+use PhpBench\Expression\PrefixParselet;
+use PhpBench\Expression\Token;
+use PhpBench\Expression\Tokens;
+
+class NameParselet implements PrefixParselet
+{
+    public function tokenType(): string
+    {
+        return Token::T_NAME;
+    }
+
+    public function parse(Parser $parser, Tokens $tokens): Node
+    {
+        $name = $tokens->chomp()->value;
+
+        return new ParameterNode([$name]);
```

Comment:
check if the `T_PARAMETER` node is really required -- looks like it can be replaced with this.